WordPress/src/main/java/org/wordpress/android/ui/prefs/SiteSettingsFragment.java

```diff
@@ -472,7 +477,16 @@ public boolean onPreferenceClick(Preference preference) {
     public boolean onPreferenceChange(Preference preference, Object newValue) {
         if (newValue == null || !mEditingEnabled) return false;
 
-        if (preference == mTitlePref) {
+        if (preference == mJpMonitorActivePref) {
+            mJpMonitorActivePref.setChecked((Boolean) newValue);
```

Comment:
Well, we could unbox at the beginning...but that would still be hideous. First world problem indeed 😆